pep-0649.rst

```diff
@@ -556,8 +583,9 @@ code should prefer to interact with it only via the
 
 The callable stored in ``__annotate__`` must accept a
 single required positional argument called ``format``,
-which will always be an ``int``.  It must either return
-a dict (or subclass of dict) or raise
+which will always be an ``int`` or an object interchangable
+with an ``int`` (e.g. ``enum.IntEnum``).  It must either
```

Comment:
I'm not sure that "interchangeable" has a precise definition in terms of Python objects. If I were to be forced to assign a precise definition to it I would probably assume it means something like Liskov substitutability? But I think we could just say " or a subclass of int (e.g. enum.IntEnum)" and that would be more clear? `IntEnum` is a subclass of int.

I suppose technically you could pass in some other thing with an `__eq__` method that is willing to compare to ints, but a) if that's really what we mean here, we should say "comparable to" not "interchangeable with", and b) why should we suggest that such a thing is supported?

Reply:
I agree.  I went with

> will always be an `int` (or a subclass of `int`).

Seems sufficient, doesn't get lost in the weeds.  Thumbs up.